app/models/squix/print_job_wrapper.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Squix
+  # Squix::PrintJobWrapper
+  class PrintJobWrapper
+    include ActiveModel::Model
+
+    attr_accessor :printer_name, :label_template_name, :label_template_id, :labels, :copies, :errors
+
+    # validate
+
+    def initialize(printer_name, label_template_name, label_template_id, labels, copies)
```

Comment:
You can remove all of this. Because the initializer will automatically do that. For copies you can set an attribute accessor method.
```
def copies=(copies)
 @copies = copies.to_i
end
```